features/onboarding/add-new-services-code/Index.feature

```diff
@@ -20,11 +20,15 @@ Feature: onboarding / dataplanes-overview / index
       """
       KUMA_DATAPLANE_COUNT: 1
       """
-    And the URL "/dataplanes" responds with
+    And the URL "/dataplanes/_overview" responds with
```

Comment:
We now check `_overview` for counts of things, I think this is fine, but let me know if not.